PythonVisualizations/SimpleSorting.py

```diff
@@ -18,12 +18,15 @@ class SimpleArraySort(VisualizationApp):
     ARRAY_Y0 = 100
     FOUND_COLOR = 'brown4'
     nextColor = 0
+    
+   
 
     def __init__(self, size=10, title="Simple Sorting", **kwargs):
         super().__init__(title=title, **kwargs)
         self.size = size
         self.title = title
         self.list = []  # Internal array of drawable cell values
+        self.animation = False # keeps track of whether or not animation is taking place

```

Comment:
Really nice idea to use the animation state as an indicator to tell if a find or insert should occur. 
However, you don't need to create a variable because we have something in VizualizationApp.py that deals with this (see below for details).

While the code works as is, by converting your code to make use of VizualizationApp.py's `animationState` variable (see below for details), the code will be fit the project's style much nicer and it will be easier for other visualizations to emulate your fix. 

Background on VizualizationApp.py's `animationState`:
At the beginning of each function we call `self.startAnimations()` which disables the buttons and calls VizualizationApp.py's `startAnimations` function. 

This is VizualizationApp.py's `startAnimations` 
```
def startAnimations(self):
        self.animationState = self.RUNNING
        self.pauseButton['text'] = 'Pause'
        self.pauseButton['command'] = lambda: self.onClick(
            self.pause, self.pauseButton)
        self.pauseButton['state'] = NORMAL
        self.stopButton['state'] = NORMAL

```

You'll see that VizualizationApp.py has a variable (`animationState`) to keep track of whether or not an animation is currently happening.  `animationState` can be `STOPPED`, `RUNNING`, or `PAUSED`.

Reply:
Michal and I made these changes and the code seems to be working properly.